sdk/python/foundation-models/system/finetune/image-instance-segmentation/mmdetection-fridgeobjects-instance-segmentation.ipynb

```diff
@@ -265,7 +265,7 @@
    "source": [
     "mmdetection_model_name = \"mask_rcnn_swin-t-p4-w7_fpn_1x_coco\"\n",
```

Comment:
the mmdetection model name in 3.x is also different, right?

Reply:
Yes, updated to "mask-rcnn_swin-t-p4-w7_fpn_1x_coco"